### DISCUSSION TITLE
Where to buy?

### DISCUSSION THREAD

**Discussion by DrDustwolf** [sh123/esp32_loradv#10] [Q&A]
Hi the project looks interesting.
Where can I buy this radio transceiver or the parts to build it?
Can you please add links?
AliExpress or eBay would be good if possible.
Otherwise if the options above don't sell it stores that ship to Germany and who are not based in the US would be good. Nearly everything you buy from the USA and ship to the EU/Germany lands at the customs office and has to be picked up manually. This us a huge waste of life time, energy and money.

**Comment by sh123** [comment]
Hi, I've got all components from Aliexpress and board was ordered from pcbway, but you will need to solder a lot of wires. Case and antenna were 3d printed. Maybe it is good idea to create new PCB board with placeholders for all modules...

**Reply by n0p** [reply]
Hi
It would help a lot to have more photos of your assembly.

**Reply by sh123** [reply]
There is complete new board design under `extras/schematics`, so no need to solder wires.

**Reply by n0p** [reply]
I know, I've ordered 75PCBs for my radioclub to build :)

But sourcing the exact component from aliexpress sometimes is a tad difficult without a few pictures. For example, I think I got the wrong encoder :)

**Reply by sh123** [reply]
In KiCad you can generate BOM bill of materials, it has all component types. I do not have specific list to be ordered right away... encoder is right angled EC11. 

75 PCBs for that toy walkie-talkie? I hope you understand that this device is not certified in any way and purely experimental and can violate some spectrum and power limit rules in some countries... So I'm not taking responsibility if you will get fine or worse for spectrum misuse.

**Reply by n0p** [reply]
Thank you!

75 PCBs because I wanted to build 2 or 3 and then colleages started getting interested in the project, so everyone wants to build a few now xD

We do know that this is experimental and we want to experiment with it :)

**Reply by n0p** [reply]
BTW; this is the BoM w/links I collected for my colleages at the amateur radio club, hope they are useful for someone:

    ESP-WROOM32 30pin - https://es.aliexpress.com/item/1005005953505528.html
    Modem LoRa E22-400M30S (SX1268) - https://es.aliexpress.com/item/1005003122961059.html
    I2S speaker module MAX98357A - https://es.aliexpress.com/item/1005005383121121.html
    4ohm 3w speaker (40mm diameter) - https://es.aliexpress.com/item/1005006684947554.html
    I2S microphone INMP441 - https://es.aliexpress.com/item/1005007889064664.html
    Rotary encoder with push button - https://es.aliexpress.com/item/1005008091948369.html 
    Small OLED display SSD1306 128×32 - https://es.aliexpress.com/item/1005008096924594.html
    Uses combined charge + 5v boost controller - https://es.aliexpress.com/item/1005008657948784.html

    18650 Cell
    Botón switch tactile SKHH angled - https://es.aliexpress.com/item/1005005369415986.html
    SMA-J-P-X-ST-EM1 Edgemount - https://es.aliexpress.com/item/10000000835065.html
    R1 - 100k 0603 - https://es.aliexpress.com/item/1005005677654015.html
    R2-R7 - 10k 0603 - https://es.aliexpress.com/item/1005005677654015.html
    C1, C3-C8, C10, C11 - 100nF 0603 - https://es.aliexpress.com/item/32371454429.html
    C2 - Elec 100uF SMD 3×5,3 - https://es.aliexpress.com/item/1005006724593986.html
    C9 - 47uF Radial D4 mm P2 mm - https://es.aliexpress.com/item/1005005022177024.html

    GPS - OPTIONAL - UNSUPPORTED ON WALKIE FW - CONFLICTS WITH ON BOARD SERIAL PORT PROGRAMMER - https://es.aliexpress.com/item/1005008733474270.html

**Reply by sh123** [reply]
@n0p, that's nice, added this list to README

**Reply by n0p** [reply]
I am not 100% sure about the caps.
One of the mods that I am thinking about doing to the PCB is bigger pads, so it will fit other sizes easily.
I soldered my units using 0805 passives

**Comment by R2AirVlad** [comment]
> Can you please add links?

You need a "narrow" ESP32 like this: [ESP32 CP2102 TYPE-C](https://www.aliexpress.us/item/3256804611058926.html?spm=a2g0o.productlist.main.67.3485205fIAiHdJ&algo_pvid=378f765c-8eda-4084-901e-39c86249bcf8&aem_p4p_detail=20231211085240386574952667960003932968&algo_exp_id=378f765c-8eda-4084-901e-39c86249bcf8-33&pdp_npi=4%40dis%21USD%213.75%213.19%21%21%213.75%21%21%402101d68d17023135599574988e7bda%2112000030522912211%21sea%21US%210%21AB&curPageLogUid=B3yROa9u0HK9&search_p4p_id=20231211085240386574952667960003932968_34)

Also you need this [modem](https://www.aliexpress.us/item/3256801312589937.html?spm=a2g0o.productlist.main.1.e8c778811DAjK6&algo_pvid=156ee164-79c7-4c6d-8326-bb3e5681617b&aem_p4p_detail=20231211085827518324469187250003517235&algo_exp_id=156ee164-79c7-4c6d-8326-bb3e5681617b-0&pdp_npi=4%40dis%21USD%217.59%216.45%21%21%217.59%21%21%402101f49a17023139070087951e99a4%2112000016357243272%21sea%21US%210%21AB&curPageLogUid=3kR9V0bLUnpI&search_p4p_id=20231211085827518324469187250003517235_1). 

I have a 3 pieces of boards free from local manufacture. Immersion gold.

This is my emplimentation of this project:
![IMG_5269](https://github.com/sh123/esp32_loradv/assets/125462390/43022710-fde2-4e50-aade-be1b153ebf3d)
![IMG_5270](https://github.com/sh123/esp32_loradv/assets/125462390/78475201-794e-47fb-b632-593a6f4ba082)

**Reply by sh123** [reply]
Wow, looks awesome!

**Reply by R2AirVlad** [reply]
Thank You! Your code is awesome also!

**Reply by uw5elk** [reply]
Hello.  I would be grateful for the stl model of the case.

**Reply by sh123** [reply]
It is under https://github.com/sh123/esp32_loraprs/tree/master/extras/cad
